src/core/Akka/Actor/TypedActor.cs

```diff
@@ -34,7 +35,10 @@ public abstract class TypedActor : ActorBase
         /// <returns>TBD</returns>
         protected sealed override bool Receive(object message)
         {
-            MethodInfo method = GetType().GetMethod("Handle", new[] {message.GetType()});
+            Type[] msgTypeArray = new[] {message.GetType()};
+            Type actorType = GetType();
+            MethodInfo method = actorType.GetMethod("Handle", msgTypeArray) ??
+                                actorType.GetMethod($"Akka.Actor.IHandle<{msgTypeArray[0].FullName}>.Handle", BindingFlags.NonPublic | BindingFlags.Instance);
```

Comment:
Yep, already did it in my local branch:
https://github.com/Szer/akka.net/commit/a1e97b9a10f0cb5ff78e4c807a5e1d5e83fc74ec#diff-b4b64856bc2f68b9523d55f258b1ba6a